fiware-region-sanity-tests/tests/fiware_region_with_networks_tests.py

```diff
@@ -160,6 +157,142 @@ def __get_external_network_test_helper__(self):
 
         return external_network_id
 
+    def __get_shared_network_test_helper__(self):
+        """
+        HELPER. Finds and returns the shared network name of current region
+        :return: Shared network name
+        """
+        shared_network_name = TEST_SHARED_NET_DEFAULT
+        shared_network_conf = self.conf[PROPERTIES_CONFIG_REGION].get(PROPERTIES_CONFIG_REGION_SHARED_NET)
+        if shared_network_conf:
+            shared_network_name = shared_network_conf.get(self.region_name, TEST_SHARED_NET_DEFAULT)
+        self.assertIsNotNone(shared_network_name, "No shared network found")
```

Comment:
This assertion should never be executed but, if it is, add as part of the message that the shared network has not been found *in configuration file*.